src/main/java/practice/StreamPractice.java

```diff
@@ -1,29 +1,47 @@
 package practice;
 
-import java.util.Collections;
+import java.util.Arrays;
+import java.util.Collection;
 import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 import model.Candidate;
+import model.Cat;
 import model.Person;
 
 public class StreamPractice {
+    private static final String EXCEPTION_MSG = "Can't get min value from list: ";
```

Comment:
Don’t  use abbreviations 
```suggestion
    private static final String EXCEPTION_MESSAGE = "Can't get min value from a list: ";
```

Reply:
Sorry my mistake